lib/Conversion/StableHLOToTTIR/StableHLOToTTIRPatterns.cpp

```diff
@@ -19,6 +19,7 @@
 #include "ttmlir/Dialect/TTIR/IR/TTIROps.h"
 
 #include <llvm/ADT/APFloat.h>
+#include <mlir/Dialect/Func/IR/FuncOps.h>
```

Comment:
Needed?

Reply:
You are correct, not needed. Fixed in the next commit.